libvips/foreign/dcrawload.c

```diff
@@ -133,8 +133,10 @@ vips_foreign_load_dcraw_set_metadata(VipsForeignLoadDcRaw *raw,
 		raw->raw_processor->idata.make);
 	vips_image_set_string(image, "raw-model",
 		raw->raw_processor->idata.model);
+#if LIBRAW_COMPILE_CHECK_VERSION_NOTLESS(0, 17)
 	vips_image_set_string(image, "raw-software",
 		raw->raw_processor->idata.software);
```

Comment:
Introduced in commit https://github.com/LibRaw/LibRaw/commit/007da06fbaef556c99d7aeb8636180b374cc029b.